state-chain/chains/src/refund_parameters.rs

```diff
@@ -72,22 +72,26 @@ pub enum AccountOrAddress<AccountId, Address> {
 	PartialOrd,
 	Ord,
 )]
-pub struct ChannelRefundParameters<A, CcmRefundDetails> {
+pub struct ChannelRefundParameters<A, CcmRefundDetails, OracleSlippage> {
```

Comment:
Same point as in the cf_parameter. If possible, avoid this and just use a field. Do we expect different types for the `OracleSlippage`?